### PR TITLE
11713 Update to all project settings to properly reference and include SwrveSDKCommon

### PR DESCRIPTION
**Pull request by seaders**

### CODE REVIEW

File: Sdk/Track/Swrve.h [hunk 1]
```diff
@@ -2,19 +2,18 @@
 #import "SwrveInterfaceOrientation.h"
 #import "SwrveReceiptProvider.h"
 #import "SwrveResourceManager.h"
-#import "SwrveSignatureProtectedFile.h"
-#import "SwrveCommon.h"
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wgnu"
+#if COCOAPODS
```
Comment (Sergio-Mira) [inline Sdk/Track/Swrve.h:6]: Why is this needed? Just curious that we didn't need if before

Comment (seaders) [inline Sdk/Track/Swrve.h:6]: The COCOAPODS flag?  SwiftProjectCocoaPods breaks if you try and reference them directly with `#import "SwrveSignatureProtectedFile.h"`, because they specifically are only available via Swift `Modules`.  Swapping the import statements to the proper angular reference, via their framework / module instead `#import <SwrveSDKCommon/SwrveSignatureProtectedFile.h>`, fixing it all up.

File: SwrveSDK.podspec [hunk 2]
```diff
@@ -14,6 +14,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Sdk/**/*.{m,h}'
   s.public_header_files = 'Sdk/**/*.h'
   s.resources = 'Sdk/Conversation/Resources/*.*'
+  s.dependency 'SwrveSDKCommon'
```
Comment (Sergio-Mira) [inline SwrveSDK.podspec]: Do we need to specify the version here?

Comment (seaders) [inline SwrveSDK.podspec]: Ah yeah, which version of Common we want, a la https://github.com/Swrve/swrve-sdk/blob/release-4_3/ios/SwrveLocationSDK/SwrveLocationSDK.podspec#L16?  Absolutely we do.  Just start at 4.2, or will we move everything to 4.3?

Comment (Sergio-Mira) [inline SwrveSDK.podspec]: We'll release them together always, so 4.3 now

Comment (seaders) [inline SwrveSDK.podspec]: doneso.

**Comment by Sergio-Mira** [comment]
:rocket:
